packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -157,6 +161,13 @@ class UnreferencedStateTracker {
     }
 }
 
+/**
+ * This sets the default container runtime expiry. We have container runtime expiry for GC purposes. Once the
+ * ContainerRuntimeExpiry is set, we should not be able to change this value. Please look at this readme #link to
+ * readme.
+ */
+ export const defaultContainerRuntimeExpiryMs = 30 * 24 * 60 * 60 * 1000; // 30 days
```

Comment:
Just noticed that this should be called defaultSessionExpiryMs. The comment should be updated accordingly.